ArchiSteamFarm/IPC/Startup.cs

```diff
@@ -133,8 +149,48 @@ internal sealed class Startup {
 			}
 		);
 
-		// Use routing for our API controllers, this should be called once we're done with all the static files mess
 #if !NETFRAMEWORK && !NETSTANDARD
+		// Add support for static files from custom plugins (e.g. HTML, CSS and JS)
+		foreach (string staticFilesDirectory in staticFilesDirectorys) {
+			app.UseStaticFiles(
+				new StaticFileOptions {
+					OnPrepareResponse = static context => {
```

Comment:
Rather than duplicating all of my logic, extract this to a standalone function and call from both places.